runners/org.osbuild.rhel90

```diff
@@ -0,0 +1,61 @@
+#!/usr/libexec/platform-python
+
+import os
+import subprocess
+import sys
+
+import osbuild.api
+
+
+def ldconfig():
+    # ld.so.conf must exist, or `ldconfig` throws a warning
+    subprocess.run(["touch", "/etc/ld.so.conf"], check=True)
+    subprocess.run(["ldconfig"], check=True)
+
+
+def sysusers():
+    try:
+        subprocess.run(["systemd-sysusers"], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, check=True)
+    except subprocess.CalledProcessError as error:
+        sys.stderr.write(error.stdout)
+        sys.exit(1)
+
+
+def tmpfiles():
+    # Allow systemd-tmpfiles to return non-0. Some packages want to create
+    # directories owned by users that are not set up with systemd-sysusers.
+    subprocess.run(["systemd-tmpfiles", "--create"], check=False)
+
+
+def nsswitch():
+    # the default behavior is fine, but using nss-resolve does not
+    # necessarily work in a non-booted container, so make sure that
+    # is not configured.
+    try:
+        os.remove("/etc/nsswitch.conf")
+    except FileNotFoundError:
+        pass
+
+
+
+def python_alternatives():
+    """/usr/bin/python3 is a symlink to /etc/alternatives/python3, which points
```

Comment:
Hmm, I've just downloaded the newest image and it points to `python3.9` directly, see:

```
$ ls -l /usr/bin/python3
lrwxrwxrwx. 1 root root 9 Feb 20 14:53 /usr/bin/python3 -> python3.9
```

Reply:
my bad, I just copied it from rhel8 runner, should've checked